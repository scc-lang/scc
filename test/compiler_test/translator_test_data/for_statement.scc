for (int from, int to = 10, int sum; from < to; from += 1) {
    sum += from;
    std::println("from 1 to {}, sum = {}", from, sum);
}