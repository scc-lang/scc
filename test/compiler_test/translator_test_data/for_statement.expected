// scc autogenerated file.

import scc.std;

int main()
{
    {
        int from {};
        int to { 10 };
        int sum {};

        for (; from < to; from += 1)
        {
            sum += from;
            scc::std::println("from 1 to {}, sum = {}", from, sum);
        }
    }
    return 0;
}
